rca/events/serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.fields import Field
+
+
+class EventTaxonomySerializer(Field):
+    def to_representation(self, value):
+        return {"title": value.title, "id": value.id}
+
+
+class RelatedDirectoratesSerializer(Field):
+    def __init__(self, related_model=None, *args, **kwargs):
```

Comment:
I can't see `related_model` being passed in any of the code in this PR, is there a use case for it?